js/fetch.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-import { assert, createResolvable, notImplemented, isTypedArray } from "./util";
+import { createResolvable, isTypedArray } from "./util";
+import { assert, unreachable, unimplemented } from "./test_util";
```

Comment:
This `unreachable` is not used. Remove it.

Reply:
Fixed some imports errors but still got the same issue :/